tools/packager/packager.py

```diff
@@ -73,6 +76,10 @@ def run_packager():
     if args.cmd == "help" or not args.cmd:
         print(USAGE)
 
+    elif args.cmd == "version":
+        p = subprocess.check_output('git describe --tags', shell=True)
```

Comment:
What if this code is exported and missing git metadata? Shouldn't we wrap this with a try/except to catch this?